config/routes.rb

```diff
@@ -351,7 +351,7 @@
       put '/hybrid_handoff' => 'hybrid_handoff#update'
       get '/link_sent' => 'link_sent#show'
       put '/link_sent' => 'link_sent#update'
-      get '/link_sent/poll' => 'capture_doc_status#show', as: :capture_doc_status
```

Comment:
Looks like there might be at least one reference to the `capture_doc_status` name that needs to be updated?

https://github.com/18F/identity-idp/blob/b32c6eb6cca97c146247005899555b4218a98963/app/views/idv/link_sent/show.html.erb#L45

Reply:
Ha, I think we found this about simultaneously. This was the only reference.